setup.cfg

```diff
@@ -32,8 +32,7 @@ packages = find:
 include_package_data = True
 python_requires = >=3.7
 install_requires =
-    pikepdf~=8.6;python_version>="3.8"
-    pikepdf==6.2.9;python_version<"3.8"
+    pypdfium2~=4.7
```

Comment:
this version is higher than our current standard

Reply:
you say that like it's a bad thing

Reply:
it is unfortunately as in the major versions, the rasterizing method changes drastically and we want to avoid those discrepancies